pkg/3scale/amp/component/apicast_monitoring.go

```diff
@@ -104,7 +104,19 @@ func ApicastPrometheusRules(ns string) *monitoringv1.PrometheusRule {
 					Name: fmt.Sprintf("%s/apicast.rules", ns),
 					Rules: []monitoringv1.Rule{
 						{
-							Alert: "ApicastRequestTime",
+							Alert: "ThreescaleApicastJobDown",
+							Annotations: map[string]string{
+								"summary":     "Job {{ $labels.job }} on {{ $labels.namespace }} is DOWN",
+								"description": "Job {{ $labels.job }} on {{ $labels.namespace }} is DOWN",
+							},
+							Expr: intstr.FromString(fmt.Sprintf(`up{job=~".*/apicast-production",namespace="%s"} == 0`, ns)),
```

Comment:
What about apicast staging? maybe adding and "or" `|`?

Reply:
Initially we agreed about not monitoring staging instance. it could raise alerts while testing configurations.

Reply:
But this is not an alert using apicast internal metrics (like 5XX errors), it is the generic prometheus scrape job alert, so it can trigger an alert if the whole apicast pod is down not offering any metric (so 3scale apicast image failing for example). It is the expected behaviour?

Reply:
makes sense for this rule, I will add it